Bookish.Web/Controllers/HomeController.cs

```diff
@@ -89,14 +99,20 @@ public IActionResult LoanBook(int copyId)
         [HttpPost]
         public IActionResult ReturnBook(int copyId)
         {
+            if (User.Claims.First().Value != bookishService.GetCopyLender(copyId))
```

Comment:
Might be worth creating a small helper method to get the current user id? I also think you should be getting it by the name of the claim, not just by assuming it's the first 🙂